build/Dockerfile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM gcr.io/distroless/static:nonroot
+FROM registry.access.redhat.com/ubi8/ubi:latest
```

Comment:
use new redhead repo

Reply:
It requires authentication, so I don't think we can do this right now.